components/toolbar/built-in-tools.md

```diff
@@ -41,7 +41,7 @@ The nested `ToolBarButton` tag exposes parameters that allow you to customize th
 | `ImageURL` | `string` | Adds an image to the button. You can provide an image url to this parameter. |
 | `SpriteClass` | `string` | Add a sprite class image to the button. Set this attribute to `k-icon MySpriteClass` where `MySpriteClass` defines the CSS rules for the sprite. |
 | `Overflow` | `ToolBarItemOverflow` enum <br /> (`Auto`) | Specifies whether the item will be hidden when the ToolBar is resized. |
-| `OverflowText` | `string` | If not specified, it will get the normal text of the item. (not valid for Separators) |
+| `OverflowText` | `string` | Defines the buttons text that will be shown only in the overflow popup. If not specified, it will get the default text of the item. (not valid for Separators) |
```

Comment:
```suggestion
| `OverflowText` | `string` | Defines the button text that will be shown only in the overflow popup. If not specified, it will get the default text of the item. (not valid for Separators) |
```